applications/asset_tracker_v2/Kconfig.sysbuild

```diff
@@ -9,10 +9,12 @@ choice BOOTLOADER
 endchoice
 
 config SECURE_BOOT
-	default y if BOARD_NRF9160DK_NRF9160_NS
+	default y if BOARD_NRF9160DK_NRF9160_NS \
+		|| BOARD_NRF9151DK_NRF9151_NS \
+		|| BOARD_NRF9161DK_NRF9161_NS
```

Comment:
what about `SOC_SERIES_NRF91X`?

Reply:
The old Thingy:91 doesn't have B0 and updatable bootloader in the static partitions. Maybe we could invert the check, so it defaults to y for SOC_SERIES_NRF91X, but `n` for the old thingy?